litmus-portal/frontend/src/components/YamlEditor/styles.ts

```diff
@@ -3,35 +3,57 @@ import { makeStyles, Theme } from '@material-ui/core';
 const useStyles = makeStyles((theme: Theme) => ({
   // Editor
 
-  statusHeading: {
+  statusHeadingInModal: {
     marginTop: theme.spacing(-2.5),
-    fontFamily: 'Ubuntu',
     fontSize: '1rem',
-    marginLeft: theme.spacing(2),
+    marginLeft: theme.spacing(2.5),
+    fontStyle: 'normal',
+    fontWeight: 500,
+    lineHeight: '130%',
+    paddingTop: theme.spacing(-4),
+  },
+
+  statusHeadingOutModal: {
+    marginTop: theme.spacing(-2.5),
+    fontSize: '1rem',
+    marginLeft: theme.spacing(2.5),
+    fontStyle: 'normal',
+    fontWeight: 500,
+    lineHeight: '130%',
+    paddingTop: theme.spacing(4),
   },
 
   statusDescription: {
-    width: '100%',
+    width: '95%',
     marginTop: theme.spacing(1.875),
-    fontFamily: 'Ubuntu',
     fontSize: '0.875rem',
     marginLeft: theme.spacing(2),
+    marginRight: theme.spacing(2),
+    fontStyle: 'normal',
+    fontWeight: 'normal',
+    lineHeight: '160%',
   },
 
   editorBackgroundFull: {
-    backgroundColor: theme.palette.common.black,
+    backgroundColor: theme.palette.editorBackground,
     color: theme.palette.secondary.contrastText,
     width: '100%',
   },
 
   horizontalLineWhite: {
     marginTop: theme.spacing(4),
-    backgroundColor: theme.palette.secondary.contrastText,
+    backgroundColor: 'rgba(255, 255, 255, 0.2)',
```

Comment:
we have a customColors in theme and have a white function there where you can pass the opacity